internal/init/init.go

```diff
@@ -1,4 +1,4 @@
-package initopts
+package init
```

Comment:
init is a problem because it is kind of a keyword. If you are going to do a pkg alias on import why now just call the pkg initPkg. Better initpkg.